scripts/make.sh

```diff
@@ -44,15 +44,10 @@ if [ -n "${OPT_BUILD}" ]; then
     go build -o ./.out/nmpolicy ./cmd/nmpolicy
 fi
 
-if [ -n "${OPT_FMT}" ]; then
-        unformatted=$(gofmt -l ./nmpolicy ./tests)
-        test -z "$unformatted" || (echo "Unformatted: $unformatted" && false)
-
-        go get golang.org/x/tools/cmd/goimports
-        unformatted=$(go run golang.org/x/tools/cmd/goimports -l --local "github.com/nmstate/nmpolicy" ./nmpolicy ./tests)
-        go mod tidy
-
-        test -z "$unformatted" || (echo "Unformatted imports: $unformatted" && false)
+if [ -n "${OPT_LINT}" ]; then
+    golangci_lint_version=v1.42.1
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin $golangci_lint_version
```

Comment:
Can you check if the file is there before downloading? It may help to run it with no internet connection.

Reply:
Done